klyshko-operator/hack/deploy.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2022 - for information on the respective copyright owner
+# see the NOTICE file and/or the repository https://github.com/carbynestack/klyshko.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+GREEN='\033[0;32m'
+NC='\033[0m' # No Color
+
+declare -a CLUSTERS=("starbuck" "apollo")
+
+echo -e "${GREEN}Undeploying from clusters${NC}"
+for c in "${CLUSTERS[@]}"
+do
+  echo -e "${GREEN}Undeploying from $c${NC}"
+  kubectl config use-context "kind-$c"
+  if [ "$c" == "apollo" ]; then
+    kubectl delete -f config/samples/klyshko_v1alpha1_tuplegenerationjob.yaml
+  fi
+  kubectl delete --all tgj
+  kubectl delete --all tgt
+  make undeploy IMG="carbynestack/klyshko-operator:v0.0.1"
+done
+
+echo -e "${GREEN}Cleaning up etcd${NC}"
+bin/etcdctl --endpoints 172.18.1.129:2379 del "/klyshko/roster" --prefix
```

Comment:
That's correct. However, the README says that it is assumed that the system is setup according to the getting started guide when using the hack scripts and that setup makes the assumption regarding the subnets as well. So changing this would be a bigger effort as it would have to be changed in the tutorials as well. I would assume that anybody configuring kind to use a different subnet would also be able to update the scripts here accordingly. Keeping that for now.